app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -1,9 +1,22 @@
-<%= content_for :before_content, govuk_breadcrumbs(breadcrumbs: {
-  "Dashboard" => dashboard_path,
-  "Your Schools" => nil
-}) %>
+<% content_for :title, 'Your schools' %>
 
-<%= render ::SchoolInformation::YourSchools::HeadingComponent.new %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% content_for :nav_bar do %>
+      <%= render PrimaryNavComponent.new do |component| %>
+        <%= component.nav_item(path: dashboard_path) do %>
+          Overview
+        <% end %>
+        <%= component.nav_item(path: lead_providers_your_schools_path) do %>
+          Schools
+        <% end %>
+      <% end %>
+    <% end %>
+  </div>
+</div>
```

Comment:
Is this right?

For one, I'm surprised the rest of the file isn't in the two divs on top (grid row and grid columns).

Also, I am pretty sure `<% content_for :nav_bar do %>` part will look the same whether it is inside of the grid divs or not, because in actuality it gets inserted outside of this html structure. 

Can we have all our `content_for`s before any page-specific html, and can we make sure those grid classes are closed in the correct place? :)

Reply:
Yes, good spot. This was just copied from another file as the work hasn't been done yet to handle this in the same way as the other navs.